src/profile-logic/import/dhat.js

```diff
@@ -191,6 +191,33 @@ export function attemptToConvertDhat(json: mixed): Profile | null {
   const otherCategory = 0;
   const otherSubCategory = 0;
 
+  // Insert a root function that is the command that was run. Note that dhat-rs includes
+  // a [root] frame, while the Valgrind implementation does not.
```

Comment:
I don't see the `[root]` frame when I import the dhat.json file from src/test/, am I missing something?